### ISSUE TITLE
About the train log of on coco dataset

### ISSUE THREAD

**Issue by ofishhaha** [ggjy/DeFeat.pytorch#18]
Thank you for ur great efforts. I met some problem when trying to reproduce ur results on coco dataset. I think it should have some difference on training settings or coefficients.
Would u mind release the train log of Faster RCNN and retinanet on COCO dataset?
It would be really helpful!

**Comment by waveboo** [comment]
The same problem so am I. And it would be wonderful if you @ggjy could provide any training configs about the COCO dataset. Thanks in advance~

**Comment by ggjy** [comment]
Sorry, I can't find the training logs now. I think the loss weight (λ) should be 4 and 16 for the foreground and background, and I use KD for backbone feature (stage 3 and 4), Neck feature, and cls-head to get the best result.
